wgpu-hal/src/vulkan/mod.rs

```diff
@@ -355,6 +355,11 @@ struct Swapchain {
     /// index as the image index, but we need to specify the semaphore as an argument
     /// to the acquire_next_image function which is what tells us which image to use.
     next_semaphore_index: usize,
+    /// The times which will be set in the next present times.
+    ///
+    /// SAFETY: This is only set if [wgt::Features::VULKAN_GOOGLE_DISPLAY_TIMING] is enabled, and
```

Comment:
```suggestion
    /// SAFETY: This is only set if [`wgt::Features::VULKAN_GOOGLE_DISPLAY_TIMING`] is enabled, and
```

Fwiw safety docs usually use `/// # Safety`, whereas `// SAFETY:` is typically used in comments.